test-support/scarb.ts

```diff
@@ -0,0 +1,7 @@
+// eslint-disable-next-line @typescript-eslint/no-require-imports
+import commandExists = require("command-exists");
+
+export const is_scarb_available = () =>
+  commandExists("scarb")
+    .then(() => true)
+    .catch(() => false);
```

Comment:
1. camelCase
2. this sounds like something we might want to cache as a global variable

```suggestion
export const scarbAvailable = commandExistsSync("scarb");
```